src/indexing/test/indexing.cpp

```diff
@@ -456,7 +456,7 @@ TEST_CASE_TEMPLATE(
 }
 
 using namespace std::chrono_literals;
-const auto max_multithread_run_time = 10s;
+const auto max_multithread_run_time = 100s;
```

Comment:
This is originally to make indexing_test pass with TSAN, but looks like somehow this PR affects to SGX Debug more.
'CCF Github CI SGX Debug' requires this timeout to be 100s, while TSAN virtual RelWithDebInfor requires only 60s.

I believe it's not an actual problem in production because SGX Release's performance doesn't seem to be affected in terms of time to pass the test. https://dev.azure.com/MSRC-CCF/CCF/_build/results?buildId=67851&view=logs&j=5435e0ac-25e5-5426-50be-61b0d0ea8d34&t=a2a8c6c1-1b13-59a7-596c-97d4f22a6857&l=401